CorsixTH/Src/CMakeLists.txt

```diff
@@ -0,0 +1,61 @@
+set(corsixth_lib_src_files 
+    ${CMAKE_CURRENT_SOURCE_DIR}/random.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/bootstrap.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/iso_fs.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/lua_rnc.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/persist_lua.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/run_length_encoder.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/sdl_audio.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/sdl_core.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/sdl_wm.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/th.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/th_gfx.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/th_gfx_font.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/th_gfx_sdl.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/th_lua.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/th_lua_anims.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/th_lua_gfx.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/th_lua_lfs_ext.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/th_lua_map.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/th_lua_movie.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/th_lua_sound.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/th_lua_strings.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/th_lua_ui.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/th_map.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/th_map_overlays.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/th_movie.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/th_pathfind.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/th_sound.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/xmi2mid.cpp
+
+    ${CMAKE_SOURCE_DIR}/common/rnc.cpp
+    PARENT_SCOPE
+)
+
+set(corsixth_lib_hdr_files
```

Comment:
I don't think the headers are actually needed. Could you check?

Reply:
To answer my own question. Yes headers should be included, they are automatically excluded from compilation, but used for IDE generators.